server/models/Profile.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const profileSchema = new Schema({
+    first_name: { type: String, required: true },
+    last_name: { type: String, required: true },
```

Comment:
I would prefer `camelCase` as it's JavaScript convention. Please change where applicable. Thanks!